.travis.yml

```diff
@@ -5,10 +5,6 @@ node_js:
 sudo: required
 dist: trusty
 
-branches:
-  only:
-    - master
```

Comment:
Did you mean to remove this line?

Removing this will cause the build to run for every branch when code is pushed to it in git.